stacks/SiriGeneratorStack.ts

```diff
@@ -11,10 +11,13 @@ export function SiriGeneratorStack({ stack }: StackContext) {
         name: `cdd-siri-sx-${stack.stage}`,
     });
 
+    const siriSXUnvalidatedBucket = new Bucket(stack, "SiriSXUnvalidatedBucket", {
+        name: `cdd-siri-sx-unvalidated-${stack.stage}`,
+    });
+
     const siriGenerator = new Function(stack, "cdd-siri-generator", {
         environment: {
```

Comment:
It would be nice to give these functions a clean name, `cdd-siri-sx-generator-${stage}` and `cdd-siri-sx-validator-${stage}`

Reply:
done

Reply:
As mentioned above, I think you've updated the bucket names rather than add in the function names, a function name can be passed in using `functionName`

Reply:
add the `functionName` field to the lambda constructs and reverted the bucket names